app/presenters/vendor_api/hesa_qualification_fields_presenter.rb

```diff
@@ -8,11 +8,11 @@ def to_hash
       return {} unless @qualification.level == 'degree'
 
       {
-        hesa_degtype: @qualification.qualification_type_hesa_code,
-        hesa_degsbj: @qualification.subject_hesa_code,
-        hesa_degclss: @qualification.grade_hesa_code,
-        hesa_degest: @qualification.institution_hesa_code,
-        hesa_degctry: @qualification.institution_country,
+        hesa_degtype: @qualification.qualification_type_hesa_code&.to_s.rjust(3, '0'),
+        hesa_degsbj: @qualification.subject_hesa_code&.to_s.rjust(6, '0'),
```

Comment:
Do we need to pad this one? It always starts with 1 or 9

Reply:
Probably not, but legit values are all 6-chars long. Worst case scenario would be vendors receiving an unrecognisable code (e.g. `000000`) if we have `0` values. If our value is `nil` they'll get `nil`, too.

Reply:
If we've got bad data no amount of padding is going to help! I think we should send what we've got. (Then fix the data in the DB).

Reply:
Ok, updated